Sources/swiftML/KMeans.swift

```diff
@@ -0,0 +1,240 @@
+import TensorFlow
+
+/// K-Means Clustering
+public class KMeans {
+    
+    public var nClusters: Int
```

Comment:
Could you please rename properties/variables representing "the number of things" to `xxCount`?

```suggestion
    public var clusterCount: Int
```

Please also update argument labels and documentation comments. 🙂